README.md

```diff
@@ -3,3 +3,9 @@
 Clone of the current Whatsapp Web UI.
 
 Built with React.
+
+To build and run this project locally:
```

Comment:
It would be better to make this a section, so this line could be converted to a heading

## Run this project locally